src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -675,16 +677,29 @@ internal static string AppendChangesToDescription(string prDescription, Build ol
                     prLink = $@"- [{comment}]({repoURL}/commit/{fullSHA})";
                 }
 
-                description.AppendLine(prLink);
+                const string limitMessage = "Changelog truncated due to description length limit.";
```

Comment:
Should we make this less verbose? "Changelog too long" or "... =("

Reply:
My vote is for "... you haven't inserted in a while, huh?"